onvm/onvm_mgr/onvm_stats.c

```diff
@@ -57,7 +57,6 @@
 
 /************************Internal Functions Prototypes************************/
 
-
```

Comment:
I think the style is 2 blank line after the /** **/ declaration (even if it isn't lets not change only a few occurences)